suites/squid/cephfs/tier-2_cephfs_test-volume-management_arch.yaml

```diff
@@ -352,3 +352,11 @@ tests:
       polarion-id: CEPH-83604079
       desc: cephfs subvolumegroup scenarios
       abort-on-fail: false
+  - test:
+      name: cephfs subvolume idempoence earmark
+      module: cephfs_vol_management.cephfs_vol_mgmt_subvol_idempotence_earmark.py
+      polarion-id: CEPH-83604184
+      desc: cephfs subvolume idempoence earmark
+      abort-on-fail: false
+      comment: "BZ2332760, BZ2332723"
```

Comment:
Corresponding code for BZ 2332723 is commented, so issue may not be seen when automation is executed. So BZ required in comment?

Reply:
one that has an issue is the earmark with "smb", i removed it to conitnues testing. should I add them and make it fail..? I was going to add additional test cases, when it is fixed.

Reply:
We dont have to add the failing scenario in this PR as merge timing is critical for master branch. Please take a note and work on it later.